store/keystore.go

```diff
@@ -199,3 +207,24 @@ func saveToPem(out io.Writer, key openssl.PrivateKey) error {
 
 	return err
 }
+
+func loadPassphrase(passphrase_file string) (passphrase string, err error) {
+	var fi *os.File
+	var dat []byte
+	if passphrase_file != "" {
+		if passphrase_file == "-" {
+			log.Debugf("Read passphrase from stdin.")
+			fi, err = os.Open("/dev/stdin")
+		} else {
+			log.Debugf("Read passphrase from file.")
+			fi, err = os.Open(passphrase_file)
+		}
+		dat, err = ioutil.ReadAll(fi)
```

Comment:
Several missing error checks. Each `err` assignment should check for `err != nil` and return error.